docs/graphql/manual/guides/integrations/auth0-jwt.rst

```diff
@@ -201,3 +201,8 @@ That’s it! This rule will be triggered on every successful signup/login and sy
 .. note::
 
    We need to use an ``upsert`` operation here because social logins do not distinguish between sign-up and login. Hence, we need to run this rule every time a successful login is made and do nothing if the user already exists.
+
+
+.. admonition:: Local dev with Auth0 rules
```

Comment:
```suggestion
.. admonition:: Note
```

Reply:
Wouldn't it be clearer to notice this if it had a bold title?

Reply:
Yes, but then it's not consistent with the rest of the docs. It always says `Note`.

Reply:
This should go into a subsection then.

Reply:
But you are right, it would be clearer.

Reply:
Let's keep it the way you proposed.

Reply:
BTW I used `admonition:: <title>` here because it allows you to supply a title. Otherwise, the rest of the doc has `::note`